### PR TITLE
Agent #9

### PR DESCRIPTION
**Pull request by mauricio1802**

### CODE REVIEW

File: Yearn-agents/ManagmentChanges/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - Fired when the event `setPerformanceFee` is emited from a Yearn Vault.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - In the metadata you can find:
+    - `vaultAddress`: Address of the Yearn Vault emiting the event.
+    - `setFee`: The new fee set.
+
+## Test Data
+
+The test of this agent use a hardhat node forked from mainnet. For running test test you'll need to create a `.env` with the following format
```
Comment (e1Ru1o) [inline Yearn-agents/ManagmentChanges/README.md:42]:
```suggestion
The test of this agent use a hardhat node forked from mainnet. For running a test you'll need to create a `.env` with the following format
```

File: Yearn-agents/ManagmentChanges/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+```
+jsonRPC=<json-rpc-endpoint>
+```
+In json RPC endpoint you'll need to set an endpoint to a node able to provide state info at block 13546171.
+After setting this run the hardhat node using `npx hardhat node` and after that run `npm test`.
```
Comment (e1Ru1o) [inline Yearn-agents/ManagmentChanges/README.md:47]: I think that you can add a command in the `package.json` for this, in order to use `npm run node` for example.

File: Yearn-agents/ManagmentChanges/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
```
Comment (e1Ru1o) [inline Yearn-agents/ManagmentChanges/package.json:4]: Update `name` & `description`

**Review by e1Ru1o — CHANGES_REQUESTED** [review]

**Review by e1Ru1o — APPROVED** [review]

**Review by rachit2501 — APPROVED** [review]

**Review by CeliktepeMurat — COMMENTED** [review]
On the other hand, I assume that this agents also should detect `StrategyAdded` event. Shouldn't it?

File: Yearn-agents/ManagmentChanges/src/agent.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  createUpdatePerformanceFeeFindingGenerator,
+  getYearnVaults,
+} from "./utils";
+import { ethers } from "ethers";
+
+const etherProvider = new ethers.providers.JsonRpcProvider(getJsonRpcUrl());
+
+export const provideHandleTransaction = (etherProvider: ethers.providers.JsonRpcProvider): HandleTransaction => {
+  return async (txEvent) => {
+    const yearnVaults: string[] = await getYearnVaults(etherProvider, txEvent.blockNumber);
```
[the remark this answers is not in the document]
Comment (mauricio1802) [reply Yearn-agents/ManagmentChanges/src/agent.ts:18]: Yes, I totally missed that 👍 .

**Comment by mauricio1802** [comment]
> On the other hand, I assume that this agents also should detect `StrategyAdded` event. Shouldn't it?

Yes, I missed it too :(.

**Review by CeliktepeMurat — APPROVED** [review]
LGTM

**Pull request merged**